trypear/pearai-app
# Remaps keybinds for creating a new chat and resuming a chat.

# What does this do?
I have remapped the key binds for creating a new chat and appending to a new chat.

Previously they were:
- Create New Chat: Cmd+L
- Append toChat: Cmd+shift+L

Now they are:
- Create New Chat: Cmd+Shift+L
- Append to Chat: Cmd+L

# How to Test
1. Launch PearAI like normal. When the Chat GUI shows up, you should see the remapped key bind on the GUI (The area that has "New Session (Cmd + L)" should now read "New Session (Cmd + Shift + L)" with all the symbols.)
2. Close the chat window
3. Press Cmd + L to resume chat window. Once this works, close chat window.
4. Press Cmd + Shift + L to create new chat window

Fixes #251
